R/map_fit.R

```diff
@@ -70,7 +70,6 @@ simulate_fit <- function(
 
 get_iov_specification <- function(model, parameters, iiv) {
   iov_specs <- attr(model, "iov")
-  if (!isTRUE(iov_specs$n_bins > 1)) iov_specs$bins <- c(0, 99999)
```

Comment:
with the latest changes in PKPDsim, this is not needed (and actually throws an unnecssary error)